core/market/decentralized/src/testing/mock_node.rs

```diff
@@ -334,10 +333,11 @@ impl MarketsNetwork {
         .await
     }
 
-    fn init_database(&self, name: &str) -> Result<DbExecutor> {
+    pub fn init_database(&self, name: &str) -> Result<DbExecutor> {
         let db_path = self.instance_dir(name);
         let db = DbExecutor::from_data_dir(&db_path, "yagna")
             .map_err(|e| anyhow!("Failed to create db [{:?}]. Error: {}", db_path, e))?;
+        db.apply_migration(crate::db::migrations::run_with_output)?;
```

Comment:
We shouldn't apply migrations here. Market applies them in internal code. This will apply them for the second time, what can potentially hide some problems, for example: If code for migrations inside market breaks, we won't know it.